pkg/virt-launcher/virtwrap/device/hostdevice/sriov/pcipool_netstatus_test.go

```diff
@@ -32,9 +32,17 @@ import (
 var _ = Describe("SRIOV PCI address pool with network-pci-map", func() {
 	var emptyFileBytes []byte
 	emptyNetworkPCIMapBytes := []byte("{}")
-	networkPCIMapWithThreeNetworks := []byte(
-		`{"network1":"0000:04:02.5","network2":"0000:04:02.7","network3":"0000:04:02.8"}`)
-
+	networkPCIMapWithThreeNetworks := []byte(`{
+		"interfaces": [
+			{"network":"pod"},
+			{"network":"network1", "deviceInfo":{
+				"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:04:02.5"}}},
+			{"network":"network2", "deviceInfo":{
+				"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:04:02.7"}}},
+			{"network":"network3", "deviceInfo":{
+				"type":"pci","version":"1.0.0","pci":{"pci-address":"0000:04:02.8"}}}
+		]
+	}`)
```

Comment:
I think `if iface.DeviceInfo != nil && iface.DeviceInfo.Pci != nil && iface.DeviceInfo.Pci.PciAddress != ""` opens up more combinations which are not covered here.

Reply:
Added unit tests to cover the mentioned cases.